context/app/static/js/components/search/Facets/DateRangeFacet.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useCallback, useState, useEffect } from 'react';
```

Comment:
```suggestion
import React, { useCallback, useState, useEffect, useMemo } from 'react';
```
To support the other suggestion on line 23